ocpp-v1_6/src/main/java/eu/chargetime/ocpp/feature/profile/ClientCoreProfile.java

```diff
@@ -223,10 +224,11 @@ public StatusNotificationRequest createStatusNotificationRequest(Integer connect
      * @param transactionId    required. The identification of the transaction.
      * @return an instance of {@link StopTransactionRequest}.
      */
-    public StopTransactionRequest createStopTransactionRequest(int meterStop, Calendar timestamp, int transactionId) {
+    public StopTransactionRequest createStopTransactionRequest(int meterStop, Calendar timestamp, Reason reason, int transactionId) {
```

Comment:
Reason wasn't part of the parameters since it's not a mandatory field. See explanation above.